features/cli/oc_set_probe.feature

```diff
@@ -196,9 +196,9 @@ Feature: oc_set_probe.feature
       | resource | pod              |
       | l        | deployment=mysql |
     Then the output should match:
-      | Readiness      |
-      | tcp-socket :33 |
-      | probe failed   |
+      | Readiness       |
+      | tcp-socket :33  |
+      | (probe failed)? |
```

Comment:
@wewang58  I guess `probe failed` exist in old versions, and does not exist in new versions. And this PR tries to fix that.

But it looks strange to me. Let me explain it a little.
Situation 1. Let's assume the string `probe failed` exist in all versions.
As it exist in all version, and it matches the output, it's a PASS.

Situation 2. Let's assume the string `probe failed` does not exist in all versions.
As we use `(xxx)?`, it means the string is optional, it's a PASS.

Situation 3. Let's assume the string `probe failed` exist in old versions, and does not exist in new versions.
As a combination of S1 and S2, we still got a PASS.

Situation 4. Let's assume the string `probe failed` exist in new versions, and does not exist in old versions.
As a combination of S1 and S2, we still got a PASS.

In total, no matter the string exist or not exist in any version, we will got a PASS. Then it's a meaningless test.
We can fix it with version check/compare, or simply remove it.

Reply:
@liangxia  removed it, please check it again, thanks